Marlin/Marlin_main.cpp

```diff
@@ -792,9 +789,12 @@ void gcode_line_error(const char* err, bool doFlush = true) {
  */
 void get_command() {
 
+  static char serial_line_buffer[MAX_CMD_SIZE];
+  static int serial_count = 0;
+
   if (drain_queued_commands_P()) return; // priority is given to non-serial commands
 
-  #if ENABLED(NO_TIMEOUTS)
+  #if defined(NO_TIMEOUTS) && NO_TIMEOUTS > 0
```

Comment:
Since `NO_TIMEOUTS` is set as an integer value we should be using `defined` here.